src/io/ResultWriter.cpp

```diff
@@ -47,19 +53,29 @@ void ResultWriter::endStep(ParticleContainer * /*particleContainer*/, DomainDeco
 	_U_pot_acc->addEntry(domain->getGlobalUpot());
 	_p_acc->addEntry(domain->getGlobalPressure());
 	if((domainDecomp->getRank() == 0) && (simstep % _writeFrequency == 0)){
-		_resultStream << simstep << "\t" << _simulation.getSimulationTime()
-			<< "\t\t" << domain->getGlobalUpot() << "\t" << _U_pot_acc->getAverage()
-			<< "\t\t" << domain->getGlobalPressure() << "\t" << _p_acc->getAverage()
-			<< "\t\t" << domain->getGlobalBetaTrans() << "\t" << domain->getGlobalBetaRot()
-			<< "\t\t" << domain->cv() << "\t\t" << domain->getglobalNumMolecules();
-		_resultStream << "\n";
```

Comment:
the only thing that you actually need to do to ensure a flush is put a std::endl here instead of a "\n".

Reply:
so you don't need all of this open and close operations ;)

Reply:
I didn't know that `endl `also flushes the stream. Why wasn't it used earlier instead of `\n` ?! ;)
Actually I didn't feel so good using open and close, so this is way better...
When only `/n` was used, the file was written at the very end and not during the execution.
I commited my newest changes already

Reply:
The automated validation checks could still fail because of the new structure of the result file...

Reply:
probably because of performance reasons, but it should be fine like this